### DISCUSSION TITLE
virus

### DISCUSSION THREAD

**Discussion by ghost** [btcsuite/btcd#2008] [General]
D:\Go\workspace\src\btcd>go install . ./cmd/...
github.com/btcsuite/btcd: go install github.com/btcsuite/btcd: open C:\Users\bravo\AppData\Local\Temp\go-build2371484554\b001\exe\a.out.exe: Operation did not complete successfully because the file contains a virus or potentially unwanted software.

**Comment by guggero** [comment]
Please don't cross post. Answer here: https://github.com/btcsuite/btcd/issues/2012
